examples/conftest.py

```
@@ -21,6 +21,59 @@
+    "gemm_streamk/test_example_tilelang_gemm_streamk.py",
+    # flash_decoding FA3 variant requires special handling
+    "flash_decoding/test_example_flash_decoding.py::test_flash_decoding_fa3",
+}
+
+
+def _match_any(nodeid, patterns):
+    """Return True if *nodeid* contains any of the *patterns*."""
+    return any(p in nodeid for p in patterns)
+
+
+def pytest_collection_modifyitems(config, items):
```

Ruff (0.15.1)

[warning] Unused function argument: `config`

(ARG001)